teams/teams-ps/teams/Get-CsTeamsShiftsConnectionSyncResult.md

```diff
@@ -0,0 +1,89 @@
+---
+external help file: Microsoft.TeamsCmdlets.PowerShell.Custom.dll-Help.xml
+Module Name: MicrosoftTeams
+title: Get-CsTeamsShiftsConnectionSyncResult
+author: gucsun
+ms.author: gucsun
+manager: navinth
+online version: https://docs.microsoft.com/powershell/module/teams/get-csteamsshiftsconnectionsyncresult
+schema: 2.0.0
+---
+
+# Get-CsTeamsShiftsConnectionSyncResult
+
+## SYNOPSIS
+
+**Note:** This cmdlet is currently in public preview.
+
+This cmdlet supports retrieving the list of user details in the mapped teams.
+
+## SYNTAX
+
+```
+Get-CsTeamsShiftsConnectionSyncResult -ConnectorInstanceId <string> -TeamId <string> [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+This cmdlet supports retrieving the list of successful and failed users in the mapped teams.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Get-CsTeamsShiftsConnectionSyncResult -ConnectorInstanceId "WCI-d1addd70-2684-4723-b8f2-7fa2230648c9" -TeamId "70f49d29-7ee1-4259-8999-946953feb79e"
+```
+```output
+FailedAadUser FailedWfmUser                 SuccessfulUser
+------------- -------------                 --------------
+{}            {FRPET, WAROS, LABRO, JOREE…} {maosha.shi@flwr0.ms, xiaoxue.chen@flwr0.ms, guchuan.sun@flwr0.ms}
+```
+
+Returns the successful and failed users in the team mapping in the instance with ID `WCI-d1addd70-2684-4723-b8f2-7fa2230648c9`.
+
+## PARAMETERS
+
+### -ConnectorInstanceId
+
+The ID of the connection instance. It can be retrieved by running [Get-CsTeamsShiftsConnectionInstance](Get-CsTeamsShiftsConnectionInstance.md)
```

Comment:
```suggestion
The ID of the connection instance. It can be retrieved by running [Get-CsTeamsShiftsConnectionInstance](Get-CsTeamsShiftsConnectionInstance.md).
```